src/client.js

```diff
@@ -72,12 +72,12 @@ const Client = class Client {
 
   _handleMessage(topic, message) {
     var parts = topic.split('/');
-    if (parts[3] !== 'activations' && parts[3] !== 'up') {
+    if (parts[4] !== 'activations' && parts[3] !== 'up') {
```

Comment:
This is not a safe check; in this context, `activations`  should only be included in `events/activations`.